### PR TITLE
fix: fix problems (volume create dp-size option rename, threshold range check, mp/dp stale paritition data)

### PR DESCRIPTION
**Pull request by Xuruida**
**What this PR does / why we need it**:
- (cli/master) changed volume create option from "size" to "dp-size"(cli) / "dpSize"(master api).
- (master) limit metanode memory threshold range check between 0.0 and 1.0 in master api.
- (metanode/datanode) check and rename local existing dp/mp directory to .old when creating new partition to avoid loading error(stale) partition data.

**Which issue this PR fixes**:

fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs-zh/source/maintenance/admin-api/master/volume.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ CubeFS以 **Owner**参数作为用户ID。
 | owner            | string | 卷的所有者，同时也是用户ID                                        | 是   | 无                                |
 | mpCount          | int    | 初始化元数据分片个数                                            | 否   | 3                                |
 | replicaNum       | int    | 副本数                                                   | 否   | 副本卷默认3（支持1,3），纠删码卷默认1（支持1-16个）   |
-| size             | int    | 数据分片大小，单位GB                                           | 否   | 120                              |
```
Comment (true1064) [inline docs-zh/source/maintenance/admin-api/master/volume.md:28]: 原来的“数据分片大小”传达意思不够准确，建议顺便改成类似“数据分片大小上限”

**Review by true1064 — DISMISSED** [review]
LGTM

File: storage/extent_store.go [hunk 2]
```diff
@@ -150,10 +152,15 @@ func NewExtentStore(dataDir string, partitionID uint64, storeSize, dpType int, i
 	s.dataPath = dataDir
 	s.partitionType = dpType
 	s.partitionID = partitionID
-	if err = MkdirAll(dataDir); err != nil {
-		return nil, fmt.Errorf("NewExtentStore [%v] err[%v]", dataDir, err)
-	}
+
 	if isCreate {
+		if err = s.backupExtent(); err != nil {
```
Comment (leonrayang) [inline storage/extent_store.go:157]: It is not necessary to rename, as it may lead to difficulties in detecting future errors.let it go.

**Review by true1064 — DISMISSED** [review]
LGTM

**Review by true1064 — APPROVED** [review]

**Review by leonrayang — APPROVED** [review]
LGTM

**Pull request merged**